ee/clickhouse/queries/test/test_breakdown_props.py

```diff
@@ -244,3 +244,51 @@ def test_breakdown_group_props(self):
         )
         result = get_breakdown_prop_values(filter, filter.entities[0], "count(*)", self.team)
         self.assertEqual(result, ["finance", "technology"])
+
+    @snapshot_clickhouse_queries
+    def test_breakdown_session_props(self):
+        _create_person(team_id=self.team.pk, distinct_ids=["p1"], properties={"$browser": "test", "$os": "test"})
+
+        # 20 second session that starts before the time range
+        _create_event(
+            team=self.team,
+            event="$pageview",
+            distinct_id="p1",
+            timestamp="2020-01-01T23:59:50Z",
+            properties={"$session_id": "1"},
+        )
+        _create_event(
+            team=self.team,
+            event="$pageview",
+            distinct_id="p1",
+            timestamp="2020-01-02T00:00:10Z",
+            properties={"$session_id": "1"},
+        )
+
+        # 70 second session
+        _create_event(
+            team=self.team,
+            event="$pageview",
+            distinct_id="p1",
+            timestamp="2020-01-02T12:00:00Z",
+            properties={"$session_id": "2"},
+        )
+        _create_event(
+            team=self.team,
+            event="$pageview",
+            distinct_id="p1",
+            timestamp="2020-01-02T12:01:10Z",
+            properties={"$session_id": "2"},
+        )
+
+        filter = Filter(
+            data={
+                "date_from": "2020-01-02T00:00:00Z",
+                "date_to": "2020-01-12T00:00:00Z",
+                "breakdown": "$session_duration",
+                "breakdown_type": "session",
+                "events": [{"id": "$pageview", "type": "events", "order": 0,}],
+            },
+        )
+        result = get_breakdown_prop_values(filter, filter.entities[0], "count(*)", self.team)
+        self.assertEqual(result, [70, 20])
```

Comment:
I assume that you'll follow up with binning results on the backend in a follow up PR?

Since on prod, this can be an insane amount of data 😅 , and doesn't make sense to bin it on the frontend - that's a CH job :P

Reply:
Yeah, exactly. That PR is still a WIP. I think we still want to support this behavior so it's consistent with other breakdowns, but agree it's totally useless